acquisitions/src/main/resources/thunderjet/mod-mosaic/init-mosaic.feature

```diff
@@ -0,0 +1,85 @@
+@parallel=false
+Feature: Initialize mod-mosaic integration tests
+
+  Background:
+    * print karate.info.scenarioName
+    * url baseUrl
+
+    * table modules
+      | name                        |
+      | "mod-permissions"           |
+      | "mod-configuration"         |
+      | "mod-login"                 |
+      | "mod-users"                 |
+      | "mod-audit"                 |
+      | "mod-orders-storage"        |
+      | "mod-orders"                |
+      | "mod-finance-storage"       |
+      | "mod-finance"               |
+      | "mod-inventory-storage"     |
+      | "mod-inventory"             |
+      | "mod-organizations-storage" |
+      | "mod-organizations"         |
+      | "mod-okapi-facade"          |
+      | "mod-mosaic"                |
+
+    * table userPermissions
+      | name                      |
+      | "mosaic.validate.get"     |
+      | "mosaic.orders.item.post" |
+
+    * table adminPermissions
+      | name                                                          |
+      | "mosaic.configuration.item.get"                               |
```

Comment:
I forgot about that earlier: please sort permissions alphabetically too. When there are duplicates, it causes an endless loop. Sorting them makes it easier to catch duplicates.